### DISCUSSION TITLE
Hide the icon

### DISCUSSION THREAD

**Discussion by AlphaNecron** [HandyOrg/HandyControl#779] [Q&A]
How do i hide the app's icon in the titlebar?

**Comment by ghost1372** [comment]
No Option, If the value of the icon property is empty, the icon will be hidden

**Reply by AlphaNecron** [reply]
But i want it to show in the taskbar...

**Reply by Yopler** [reply]
You can put this in your .csproj : 
`        
<ApplicationIcon>favicon.ico</ApplicationIcon>
`
Your project will have an Icon in the taskbar and you don't need to declare it in `<Window/>`
